src/containers/Flow/FlowList/FlowList.test.tsx

```diff
@@ -21,15 +22,17 @@ import { Flow } from '../Flow';
 import { getFilterTagQuery } from 'mocks/Tag';
 import { getRoleNameQuery } from 'mocks/Role';
 
+const isActiveFilter = { isActive: true, isTemplate: false };
+
 const mocks = [
-  getFlowCountQuery,
-  getFlowCountQuery,
-  getFlowCountQuery,
-  getFlowCountQuery,
-  filterFlowQuery,
-  filterFlowQuery,
-  filterFlowQuery,
-  filterFlowQuery,
+  getFlowCountQuery(isActiveFilter),
```

Comment:
At this point can you check why there are multiple calls being made to filter and count query and see if you can reduce them. If it seems to take more time, do it in a separate PR